services/user.py

```diff
@@ -0,0 +1,55 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> User:
+    new_user = User.objects.create_user(
```

Comment:
Use `get_user_model` instead of `User` here and in the other places as well as mentioned in the checklist